_test.py

```diff
@@ -17,6 +17,7 @@
 if __name__ == "__main__":
   parser = argparse.ArgumentParser()
   parser.add_argument("-clean", help="clean run, as if run for the first time", action="store_true")
+  parser.add_argument("-single_threaded", help="run tests in single threaded mode", action="store_true")
```

Comment:
Is this only for the tests?
There is a define within rex called : REX_SINGLE_THREADED I was planning on configuring this macro via the build system. this will probably conflict with that. 
This is not something you should fix on this branch just keep it in mind, maybe we should create an issue for this.

Reply:
this is different, the single threaded is for the python scripts, it's not for rex

Reply:
yes, it shouldn't be fixed in this branch, I thought it was fixed in async_ci

Reply:
That's what I said that it's different, but it's going to conflict

Reply:
then just rename the param in _test.py, I just need to know because teamcity uses it